Validator - Watchtower, expecations, more (#239)
helium/docs: docs/mine-hnt/validators/validators.mdx

- Join the [Helium Discord](https://discord.gg/helium) server to take part in the real-time Validator development. Participate in the `#validator` channel, but more importantly subscribe to alerts in `#validators-announcements`. `#validators-announcements` will provide you with important, timely information for upgrades and feedback during this process.
- Be attentive to upgrade the Validator software when requested through `#validators-announcements`. This may include tasks that will require you to delete & re-stake your Validator instance, or perform other tasks that will assist the team in bringing the best product to the network.
- Be proficient in command-line driven linux-based operating systems, comfortable working with cloud based service providers (AWS, Azure, Linode, DigitalOcean, or GCP, etc.), comfortable with working with firewall rules / NAT and other basic networking components, and experienced in working with docker images and controls.

The Helium team as well as the greater Helium community is continually attemping to make this process as clear and easy to understand as possible. However, during this time you can expect much change.

:::warning

> If you do not feel you can meet these expectations, thats fine! It may be better to wait for the public availablity of validators. There will be future announcements on the availability of staking your own validator on mainnet.

Reviewer: ```suggestion
We understand that not everyone will be able to meet these expectations. It may be better to wait for the public availability of validators. There will be future announcements on the availability of staking your own validator on mainnet.
```

Softer tone in the first sentence and a typo fix.
